scripts/generate-credentials.sh

```diff
@@ -135,4 +135,10 @@ DEFAULT_REALM=gather
 REALM_COOKIE=gather-realm
 MULTITENANCY=yes
 # ------------------------------------------------------------------
+
+# ------------------------------------------------------------------
+# Consumer
+# ==================================================================
+ES_CONSUMER_URL=http://gather.local:5601
```

Comment:
_Growing Pains II part_ :thinking: 

Do we need to set up ES in Gather ??? :scream:

Reply:
This is needed to `activate`/`deactivate` the ES consumer

Reply:
I knew it, I meant if we need to include all the ES platform to test this new feature... including Kafka&co, aether-producer, es-consumer... :sweat: